bitex/interface/bittrex.py

```diff
@@ -42,49 +42,43 @@ def _get_supported_pairs(self):
     @check_and_format_pair
     def ticker(self, pair, *args, **kwargs):
         """Return the ticker for the given pair."""
-        payload = {'market': pair}
-        payload.update(kwargs)
-        return self.request('public/getmarketsummary', params=payload)
+        kwargs.update({'market': pair})
+        return self.request('public/getmarketsummary', params=kwargs)
 
     @check_and_format_pair
     @format_with(BittrexFormattedResponse)
     def order_book(self, pair, *args, **kwargs):
         """Return the order book for the given pair."""
-        payload = {'market': pair, 'type': 'both'}
-        payload.update(kwargs)
-        return self.request('public/getorderbook', params=payload)
+        kwargs.update({'market': pair, 'type': 'both'})
+        return self.request('public/getorderbook', params=kwargs)
```

Comment:
This may not work - since this includes ALL kwargs passed, it would also send the ones not supported by bittrex, wich is going to result in a faulty request.